core/src/main/java/org/mskcc/cbio/oncokb/bo/impl/AlterationBoImpl.java

```diff
@@ -351,6 +351,13 @@ public void deleteMutationsWithoutEvidenceAssociatedByGene(Gene gene) {
             }
         }
 
+        if (addVUSMutation(alteration)){
+            Alteration VUSMutation = findAlteration("VUS Mutations", fullAlterations);
+            if (VUSMutation != null){
+                alterations.add(VUSMutation);
+            }
+        }
```

Comment:
Let's put this above `addOncogenicMutations`
The order of the relevant alterations matters. And it should before Oncogenic Mutations